### PR TITLE
Chore/cherrypicks 20220727

### PR DESCRIPTION
**Pull request by francisco-nery**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -103,26 +103,27 @@
...
+        <kotlin.version>1.7.10</kotlin.version>
         <junit.version>4.12</junit.version>
         <muleWsdl4j.version>1.6.4</muleWsdl4j.version>
         <slf4j.version>1.7.29</slf4j.version>
         <apache.commons.lang3.version>3.12.0</apache.commons.lang3.version>
-        <cxfRtWsdlVersion>3.4.4</cxfRtWsdlVersion>
+        <cxfRtWsdlVersion>3.5.3</cxfRtWsdlVersion>
         <hamcrest.version>2.2</hamcrest.version>
-        <mule.metadata.version>1.2.4</mule.metadata.version>
+        <mule.metadata.version>1.5.0-SNAPSHOT</mule.metadata.version>
```
Comment (acurci-at-mulesoft) [inline pom.xml:113]: Be careful. Did you check if is accurate to change libraries to manage the metadata in a support branch? What changes were introduced in the previous minor versions?

File: pom.xml [hunk 2]
```diff
@@ -103,26 +103,27 @@
     </developers>
 
     <properties>
-        <kotlin.version>1.5.21</kotlin.version>
+        <kotlin.version>1.7.10</kotlin.version>
```
Comment (acurci-at-mulesoft) [inline pom.xml:106]: Please, check if kotlin 1.7 works fine with java 8

File: pom.xml [hunk 3]
```diff
@@ -103,26 +103,27 @@
...
         <apache.commons.lang3.version>3.12.0</apache.commons.lang3.version>
-        <cxfRtWsdlVersion>3.4.4</cxfRtWsdlVersion>
+        <cxfRtWsdlVersion>3.5.3</cxfRtWsdlVersion>
         <hamcrest.version>2.2</hamcrest.version>
-        <mule.metadata.version>1.2.4</mule.metadata.version>
+        <mule.metadata.version>1.5.0-SNAPSHOT</mule.metadata.version>
         <kluentVersion>1.68</kluentVersion>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.enforcer.plugin.version>1.4.1</maven.enforcer.plugin.version>
+        <maven.enforcer.plugin.version>3.1.0</maven.enforcer.plugin.version>
```
Comment (acurci-at-mulesoft) [inline pom.xml:116]: we are doing an upgrade for two major versions ahead. did you check the RNs?

**Review by acurci-at-mulesoft — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (francisco-nery) [reply pom.xml:106]: Yes it works fine.  Per docs: By default, the Kotlin/JVM compiler produces Java 8 compatible bytecode.

**Review by acurci-at-mulesoft — APPROVED** [review]

**Pull request merged**